pkg/model/awsmodel/iam.go

```diff
@@ -124,26 +125,64 @@ func (b *IAMModelBuilder) Build(c *fi.ModelBuilderContext) error {
 		}
 	}
 
+	iamSpec := b.Cluster.Spec.IAM
+	if iamSpec != nil {
+		for _, sa := range iamSpec.ServiceAccountExternalPermissions {
+			var p *iam.Policy
+			aws := sa.AWS
```

Comment:
We might consider handling `aws == nil` to save overlooking when someone adds a second external system.